### PR TITLE
Update to 1.3

### PR DESCRIPTION
**Pull request by asdftamir**

### CODE REVIEW

File: helm/seekret/values.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ injector:
 # Minio values
 s3:
   bucketName: seekret
-  folderName: pcaps/
+  folderName: pcaps
```
Comment (val06) [inline helm/seekret/values.yaml:14]: are you sure it is not pcaps/ ?

**Review by val06 — CHANGES_REQUESTED** [review]
cr-ed
also change the docker-compose.yaml and the readme

File: helm/seekret/values.yaml (hunk 1 above)
Comment (asdftamir) [reply helm/seekret/values.yaml:14]: yup

File: dockers/README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ _Bucket access parameters: (**or** use an IAM role **instead**)_
 - AWS_SECRET_KEY        - secret key as AWS generated
 
 - BUCKET_NAME           - customer_name
-- FOLDER_NAME           - pcaps/container_name/
+- FOLDER_NAME           - pcaps/container_name
```
Comment (val06) [inline dockers/README.md:22]: fix the running line as well (it says currently `sniffer:1`

**Review by val06 — CHANGES_REQUESTED** [review]
recred

**Review by val06 — APPROVED** [review]

**Pull request merged**